test/appium/tests/atomic/account_management/test_wallet_management.py

```diff
@@ -146,3 +146,49 @@ def test_deny_camera_access_scanning_wallet_adders(self):
         send_transaction.chose_recipient_button.click()
         send_transaction.scan_qr_code_button.click()
         send_transaction.deny_button.wait_for_visibility_of_element(2)
+
+    @marks.testrail_id(3730)
+    def test_filter_transactions_history(self):
```

Comment:
I thought, why not use some existing account here for this test? With the pre-set transactions. 
It's fine to have like it is now, but i'd say it might be faster to recover account and navigate right to Transaction History screen for checks.

Reply:
makes sense, I'll redo it